### PR TITLE
fix(Tooltip): added min width

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## What?

- Added `min-width` to `Tooltip`

## Why?

- So it not looks too narrow on mobile
Screenshot 2021-11-10 at 15 49 38
Screenshot 2021-11-10 at 15 47 56

_Referenced ticket(s): [GRW-565](https://hedvig.atlassian.net/browse/GRW-565)_

### CODE REVIEW

File: src/client/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -84,11 +84,11 @@ const TooltipText = styled.p`
...
 export const Tooltip: React.FC<TooltipProps> = ({ body }) => {
-  const ref = React.useRef<HTMLDivElement>(null)
-  const [isVisible, setVisible] = React.useState(false)
+  const ref = useRef<HTMLDivElement>(null)
+  const [isVisible, setVisible] = useState(false)
 
-  React.useEffect(() => {
-    if (isVisible === false) return
+  useEffect(() => {
+    if (!isVisible) return
```
Comment (robinandeer) [inline src/client/components/Tooltip/Tooltip.tsx:91]: nit: is this really more readable than what we had before? ;)

**Review by robinandeer — APPROVED** [review]
Nice addition!

File: src/client/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply src/client/components/Tooltip/Tooltip.tsx:91]: 🤔 I think so. I mean, it's almost like reading plain english: "if not visible". But I can revert it, no problem at all.

Comment (robinandeer) [reply src/client/components/Tooltip/Tooltip.tsx:91]: aha interesting 😅 I read it as "if not _is_ visible" which makes me confused -- while the other one is "if is visible is false". But don't change it unless other people feel the same as me hehe

Comment (gustaveen) [reply src/client/components/Tooltip/Tooltip.tsx:91]: I think when we use the `is` prefix for booleans, I have no trouble reading it like `!isVisible`. So I approve 🤗

**Review by gustaveen — APPROVED** [review]
Good job! 🙌

**Pull request merged**